OIPA/iati/parser/IATI_2_03.py

```diff
@@ -4457,6 +4457,7 @@ def iati_activities__iati_activity__result__indicator__period__actual__comment__
 
         return element
 
+    # TODO : This testing is posponed on 16/102018 due to a possible bug
```

Comment:
So I was complaining that you referenced issue by line number (because line numbers can change over time), but then you completely removed the reference :D... I think you should add `see: #<issue number>` here, because if you comment something like "this is postponed and will be fixed", this essentially tells nothing for other people reading the code in the future :)